cypress/support/commands.js

```diff
@@ -0,0 +1,34 @@
+// ***********************************************
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+
+import { login } from './commands/login.js';
+
+//Setup user data
+let users = require('../fixtures/accounts.json');
```

Comment:
always use const.
That's why having constants is easier, this why you can have a better scope when you import etc.